.github/workflows/lint.yaml

```diff
@@ -8,5 +8,7 @@ jobs:
         - uses: actions/setup-node@v2
           with:
             node-version: 22
+            cache: yarn
+        - run: yarn set version stable
```

Comment:
If it doesn't work, maybe the `Corepack` should be enabled

Reply:
it's working now, I found the issue (https://github.com/curvefi/curve-frontend/pull/380#discussion_r1790269777)